util.ts

```diff
@@ -51,3 +51,19 @@ export function createResolvable<T>(): Resolvable<T> {
   });
   return Object.assign(promise, methods) as Resolvable<T>;
 }
+
+/* tslint:disable:no-any */
+export function once(fn: Function) {
```

Comment:
I'd rather call this function `onceStrict`. (because it throws error when you call it more than once)